### PR TITLE
Improved Guideline Documentations

### PR DESCRIPTION
**Pull request by int0x81**
# New Pull Request

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

Please make sure you can check the following boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [x] All new and existing tests passed

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes

### Description

Improved and reviewed the documentation in the guideline section

### CODE REVIEW

File: docs/tutorials/15-share-dependencies.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ The diagram below shows how this works. Every pilet that uses import maps talks
 
 That way you can safely use multiple versions of your dependencies. If you consolidate them you'll get a performance improvement. Otherwise, you'll get the performance penalty as usual, however, you gained the flexibility to independently choose your version.
 
-The effort to actually use import maps is quite low. The first step is to declare potentially shared dependencies via the pilet's *package.json*:
+The effort to actually use import maps is quite low. The first step is to declare potentially shared dependencies via the pilets *package.json*:
```
Comment (FlorianRappl) [inline docs/tutorials/15-share-dependencies.md:134]: Not sure why it has been changed here, too. A "pilet" and the "pilet's thing". Its like a "person" and a "person's thing".

File: docs/tutorials/04-the-pilet-api.md [hunk 2]
```diff
@@ -12,21 +12,21 @@ So far we almost exclusively used existing elements without deeply understanding
...
+We also have a video tutorial:
 
 @[youtube](https://youtu.be/s8dYsd7eQPc)
 
 ## The Pilet Foundation
 
 As described earlier a pilet is just a library compiled as an UMD module that exposes a function called `setup`. The `setup` function receives a special object that we like to call the *pilet API*. It is the API *created specifically for the pilet*. This allows the pilet to bring in new functionality to the application shell (called Piral instance).
 
-The application shell determines how the pilet API looks like. In Piral we already determine some common parts for the pilet API - things like an API to register pages, dashboard tiles, or modal dialogs. Furthermore, some convenience functions may also be included in the pilet API. For instance, in Piral we've added functions for handling the pilet's global state.
+The application shell determines how the pilet API looks like. In Piral we already determine some common parts for the pilet API - things like an API to register pages, dashboard tiles, or modal dialogs. Furthermore, some convenience functions may also be included in the pilet API. For instance, in Piral we've added functions for handling the pilets global state.
```
[the remark this answers is not in the document]
Comment (int0x81) [reply docs/tutorials/04-the-pilet-api.md]: adapted

File: docs/tutorials/15-share-dependencies.md (hunk 1 above)
Comment (int0x81) [reply docs/tutorials/15-share-dependencies.md]: adatped

File: docs/tutorials/04-the-pilet-api.md [hunk 3]
```diff
@@ -45,13 +45,13 @@ The API contains functionality to be placed in the following categories:
...
 
 ## Adding a Page
 
-While the pilet API gives us a powerful and flexible way to describe what to do with our pilet, it also leads to complexity. Alternatively, we could think of describing what pages (or components in general) to use in the *package.json*. While such a method may have some advantages, it will always miss the flexibility that we gain by using code to perform the registration.
+While the pilet API gives us a powerful and flexible way to describe what to do with our pilet, it also leads to complexity. Alternatively, we could think of describing what pages (or components in general) to use in the package.json. While such a method may have some advantages, it will always miss the flexibility that we gain by using code to perform the registration.
 
-Assuming we create a new pilet for the previously created application shell `my-app` we would end up with the following root module:
+Assuming we create a new pilet for the previously created application shell my-app we would end up with the following root module:
```
[the remark this answers is not in the document]
Comment (int0x81) [reply docs/tutorials/04-the-pilet-api.md]: adapted

**Pull request merged**